ansible/vars/flatcar/flatcar.yaml

```diff
@@ -3,6 +3,7 @@ containerd_sha256: "2697a342e3477c211ab48313e259fd7e32ad1f5ded19320e6a559f50a82b
 
 ansible_python_interpreter: /opt/bin/python
 python_path: /opt/bin/builder-env/site-packages
+containerd_cri_socket: /run/docker/libcontainerd/docker-containerd.sock
```

Comment:
This is needed for flatcar non GPU to function - let's land it!

Reply:
Broke it with https://github.com/mesosphere/konvoy-image-builder/commit/8cf3a5ffbeecae2a9c4ad141834007975a476fbe#diff-6cd7dde3b0707c4663c2e23028d4da79899f916586421e3364621480207aaf71